common/templates/ellipses.html

```diff
@@ -1,18 +1,18 @@
 <td ng-if="config.viewable || config.editable || config.deletable || config.selectMode != 'no-select'" class="block actions-layout">
     <div class="btn-group" style="display: flex">
-        <a ng-if="config.viewable" type="button" class="view-action-button btn btn-xs btn-default btn-no-focus" ng-href="{{viewLink}}" tooltip-placement="bottom" uib-tooltip="View">
+        <a ng-if="config.viewable" type="button" class="view-action-button btn btn-xs btn-primary btn-no-focus" ng-href="{{viewLink}}" tooltip-placement="bottom" uib-tooltip="View Details">
             <span class="glyphicon glyphicon-eye-open"></span>
         </a>
-        <button ng-if="config.editable && tuple.reference.canUpdate" type="button" class="edit-action-button btn btn-xs btn-default btn-no-focus" ng-click="edit()" target="_blank" tooltip-placement="bottom" uib-tooltip="Edit">
+        <button ng-if="config.editable && tuple.reference.canUpdate" type="button" class="edit-action-button btn btn-xs btn-primary btn-no-focus" ng-click="edit()" target="_blank" tooltip-placement="bottom" uib-tooltip="Edit">
             <span class="glyphicon glyphicon-pencil"></span>
         </button>
-        <button ng-if="config.deletable && unlink && associationRef.canDelete" type="button" class="delete-action-button btn btn-xs btn-default btn-no-focus" ng-click="unlink()" tooltip-placement="bottom" uib-tooltip="Unlink">
+        <button ng-if="config.deletable && unlink && associationRef.canDelete" type="button" class="delete-action-button btn btn-xs btn-primary btn-no-focus" ng-click="unlink()" tooltip-placement="bottom" uib-tooltip="Unlink">
             <span class="glyphicon glyphicon-remove-circle"></span>
         </button>
-        <button ng-if="config.deletable && delete && tuple.reference.canDelete" type="button" class="delete-action-button btn btn-xs btn-default btn-no-focus" ng-click="delete()" tooltip-placement="bottom" uib-tooltip="Delete">
-            <span class="glyphicon glyphicon-trash" style="color: black"></span>
+        <button ng-if="config.deletable && delete && tuple.reference.canDelete" type="button" class="delete-action-button btn btn-xs btn-primary btn-no-focus" ng-click="delete()" tooltip-placement="bottom" uib-tooltip="Delete">
+            <span class="glyphicon glyphicon-trash" style="color:white"></span>
```

Comment:
`.glyphicon-trash {
    color: #428bca;
}`

`button .glyphicon-trash {
    color: #fff;
}`

**These classes are maintained in `app.css`.**